bor/handler.go

```diff
@@ -29,15 +30,22 @@ func HandleMsgProposeSpan(ctx sdk.Context, msg sdk.Msg, k Keeper) sdk.Result {
 	var proposeMsg types.MsgProposeSpanV2
 	switch msg := msg.(type) {
 	case types.MsgProposeSpan:
+		if ctx.BlockHeight() >= helper.GetAntevortaHeight() {
+			k.Logger(ctx).Error("Msg span is not allowed after hardfork height")
+			return sdk.ErrTxDecode("Msg span is not allowed after hardfork height").Result()
+		}
 		proposeMsg = types.MsgProposeSpanV2{
 			ID:         msg.ID,
-			Proposer:   msg.Proposer,
```

Comment:
Hmm could we keep this field ? For logging purposes and stuff ?

Reply:
Same applies for the side handler.

Reply:
I removed it because of linter but I can return and log these values, good idea.

Reply:
Fixed.